css/utilities/background.css

```diff
@@ -0,0 +1,31 @@
+.bg-add-cart {
+  background: var(--COLOR-ADD-CART);
+}
+
+.bg-add-cart-hover:hover {
```

Comment:
why not just `.bg-add-card:hover`? or `.bg-red-500:hover` ?

Reply:
These are for those cases when I want `red-500` on hover only. And same for `add-cart`.

Like suppose if a button has a `red-400` background, and on hover, I want to change the background to `red-500`, so if I will do `bg-red-500:hover` then first of all I would have to give class `bg-red-500` to the button which will change its default background from `red-400` to `red-500`.

In my code, I have used it at this place 
![image](https://user-images.githubusercontent.com/56081584/152342372-ebd1d651-50d8-4c3e-a513-def323e87dc2.png)

Here, `btn-error-solid` has a `red-500` background normal and `red-600` on hover, but I wanted to keep `red-500` on hover as well, that's why I used. And I could also use `bg-red-500` directly as you suggested but then if I will want `red-500` on hover at other place then I will have to use `red-500` for normal too.

Reply:
Got it. That's a good idea.